src/components/templates/Jazzbar/JazzTab/JazzTab.scss

```diff
@@ -63,20 +90,27 @@
         flex-direction: column;
         align-items: center;
         background-color: rgb(25, 24, 26);
-        width: 80%;
+        width: 90%;
```

Comment:
Where does this `90%` come from?

Reply:
Better visual interception. It was done to get as much space for videos as possible. By making `actions container` larger, we took the unused space under the video

Reply:
Fair enough, makes sense.